inductiva/resources/README.md

```diff
@@ -1,17 +1,21 @@
 # Manage Computational Resources
 
-By default, simulation requests are processed by a shared pool of machines serving multiple users. If you require dedicated resources, the **Inductiva API** allows you to easily set up virtual machines reserved solely for your simulations. These are managed via machine groups, *i.e.*, groups of homogeneous machines with specific requested properties that can be started and terminated on demand. With the `MachineGroup` class, users can configure, start, and terminate machines. Creating a `MachineGroup` requires specifying the type of Virtual Machine to use and the number of such VMs that will compose the group. Currently, the available options for the `machine_type` are the ones available in the [Google Cloud Platform](https://cloud.google.com/compute/docs/machine-types). Once a `MachineGroup` is created, simply pass it as argument to your simulations, which will then be scheduled to run on those machines. Note that a `MachineGroup` is literally a group of individual machines that do not communicate with each other. In other words, a `MachineGroup` is not a computational cluster where the load of each simulatin is divided over all machines of the cluster
+By default, simulation requests are processed by a shared pool of machines serving multiple users. If you require dedicated resources with custom configurations, the **Inductiva API** allows you to easily set up virtual machines reserved solely for your simulations. These are managed via machine groups, *i.e.*, groups of homogeneous machines with specific requested properties that can be started and terminated on demand. With the `MachineGroup` class, you can configure, start, and terminate machines. Creating a `MachineGroup` requires specifying the type of Virtual Machine to use and the number of such VMs that will compose the group. Currently, the available options for the `machine_type` are the ones available in the [Google Cloud Platform](https://cloud.google.com/compute/docs/machine-types). Once a `MachineGroup` is created, simply pass it as argument to your simulations, which will then be scheduled to run on those machines. 
```

Comment:
I think we should state the type of machine/cores available in this shared default pool.